core/runtime/src/main/java/io/quarkiverse/jberet/runtime/JBeretConfig.java

```diff
@@ -31,6 +33,12 @@ public interface JBeretConfig {
      */
     Repository repository();
 
+    /**
+     *
+     */
+    @ConfigProperty(name = "max-async")
+    Optional<String> maxAsync();
```

Comment:
No need to use `@ConfigProperty`, and this can be mapped directly to an `int` or `Integer`.

Reply:
Adapted accordingly